README.md

```diff
@@ -67,6 +67,7 @@ What kind of metadata can you convey using badges?
 ## Services using the Shields standard
 * [Badger](https://github.com/badges/badgerbadgerbadger)
 * [badges2svg](https://github.com/bfontaine/badges2svg)
+* [Codacy](https://www.codacy.com)
```

Comment:
Could you remove this line please? This list is meant for entities external to shields.io.